### ISSUE TITLE
[Feature Request] Unload web panel on clicking close button

### ISSUE THREAD

**Issue by emvaized** [Floorp-Projects/Floorp#359]
### Is your feature request related to a problem?

When I click close button in the corner of web panel, panel gets closed, but not actually unloaded — when I open it again, it turns out to be running in background and keeps it's state and url. There's entry in panel's context menu to unload it, but it's not handy to use it all the time.

### Describe the solution you'd like

Would be great to have an option to unload the panel automatically when user clicks the close button. 

And for cases when user actually needs to keep panel running in background (e.g. minimize it), preserve current behavior for click on panel's icon — panel will be minimized, but not unloaded. 

### Describe alternatives you've considered

Maybe also makes sense to consider adding separate "minimize panel" button, and indicate somehow that panel is minimized (maybe show some little blue dot in the bottom right corner of the icon?)

**Comment by surapunoyousei** [comment]
This is useful when playing music in the background, so I am specifying this. It would just be nice to have the option.

**Comment by emvaized** [comment]
> useful when playing music in the background

I agree. That's why I suggested preserving the current behavior when clicked on the icon itself. But I'm 100% okay with this being an option in settings.
